src/Common/MongoProxyClient.test.ts

```diff
@@ -5,7 +5,14 @@ import { Collection } from "../Contracts/ViewModels";
 import DocumentId from "../Explorer/Tree/DocumentId";
 import { extractFeatures } from "../Platform/Hosted/extractFeatures";
 import { updateUserContext } from "../UserContext";
-import { deleteDocument, getEndpoint, getFeatureEndpointOrDefault, queryDocuments, readDocument, updateDocument } from "./MongoProxyClient";
+import {
```

Comment:
looks like unintentional changes cos you touched the file by accident and the formatter kicked in, please revert

Reply:
replaced master